CONTRIBUTING.md

```diff
@@ -652,27 +652,33 @@ Refer datalad/config.py for information on how to add these environment variable
 - `update-changelog`: uses above `linkissues-changelog` and updates .rst changelog
 - `release-pypi`: ensures no `dist/` exists yet, creates a wheel and a source distribution and uploads to pypi.
 
-## Changelog section
-
-For the upcoming release use this template
-
-```markdown
-## 0.14.6 (??? ??, 2021) -- will be better than ever
-
-bet we will fix some bugs and make a world even a better place.
-
-### Major refactoring and deprecations
-
-- hopefully none
-
-### Fixes
-
-?
-
-### Enhancements and new features
-
-?
-```
+## Releasing with GitHub Actions, auto, and pull requests
+
+New releases of datalad are created via a GitHub Actions workflow built
+around [`auto`](https://github.com/intuit/auto).  Whenever a pull request is
+merged into `maint` that has the "`release`" label, `auto` updates the
+changelog based on the pull requests since the last release, commits the
+results, tags the new commit with the next version number, and creates a GitHub
+release for the tag.  This in turn triggers a job for building an sdist & wheel
+for the project and uploading them to PyPI.
+
+### Labelling pull requests
+
+The section that `auto` adds to the changelog on a new release consists of the
+titles of all pull requests merged into master since the previous release,
+organized by label.  `auto` recognizes the following PR labels:
+
+- `major` — for changes corresponding to an increase in the major version
+  component
```

Comment:
I do not see us using it in foreseeable future ;-) lets just remove for now
```suggestion
```
and I will remove the label itself -- do you think it would cause `auto` some indigestion @jwodder?

Reply:
No.